core/templates/domain/exploration/ParamSpecsObjectFactory.ts

```diff
@@ -101,10 +101,14 @@ export class ParamSpecs {
    *    consumption.
    */
   toBackendDict(): ParamSpecsBackendDict {
-    var paramSpecsBackendDict = {};
-    this.forEach((paramName, paramSpec) => {
-      paramSpecsBackendDict[paramName] = paramSpec.toBackendDict();
-    });
+    var paramSpecsBackendDict: ParamSpecsBackendDict = {};
+    this.forEach(
+      (
+          paramName: string | number,
```

Comment:
Why `paramName` can be `number`?

Reply:
Thanks for catching this,it was some mistake.